collectors/COLLECTORS.md

```diff
@@ -417,6 +427,8 @@ The Netdata Agent can collect these system- and hardware-level metrics using a v
     application using the `apps.plugin` collector.
 -   [systemd](/collectors/cgroups.plugin/README.md): Monitor the CPU and memory usage of systemd services using the
     `cgroups.plugin` collector.
+-   [systemd unit states](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/systemdunits): Get
+    metrics on the states of services, sockets, devices, paths, and much more.
```

Comment:
They are [systemd unit types](https://www.digitalocean.com/community/tutorials/understanding-systemd-units-and-unit-files#types-of-units). 

> Get systemd units active state.

^^ or something